packages/shared/clickhouse/migrations/unclustered/0032_add_dataset_version_to_dataset_run_items_rmt.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE dataset_run_items_rmt ADD COLUMN IF NOT EXISTS dataset_version Nullable(DateTime64(3));
```

Comment:
Consider adding `AFTER` as well in case you want to place it next to another specific column.